public/javascripts/agree.js

```diff
@@ -0,0 +1,77 @@
+function Agreement() {
+  this.$inputWrapper = document.querySelector(".input-wrapper");
+  this.$allAgreeInput = document.getElementById("all-agree");
```

Comment:
```suggestion
function Agreement($agreement) {
  this.$inputWrapper = $agreement.querySelector(".input-wrapper");
  this.$allAgreeInput = $agreement.querySelector(".all-agree");
```

컴포넌트화를 시키는 차원에서 생성자 인자로 해당 컴포넌트 템플릿 dom을 넣어주고 그 안에서 필요한 element들을 선택하는게 어떠할까요?

id의 경우에서 모두 class로 바꿔서 혹시 모들 다른 코드와의 중복을 피하고요.

Reply:
현재 `Agrement` 생성자는 가입•약관 페이지에 국한된 기능만을 수행하고 있는데

컴포넌트화를 어떤 식으로 진행할 수 있을까요? 🙃

Reply:
좋은 의견인 것 같습니다.

관련 사항 적용해서 수정 후 커밋 하겠습니다~~ 😋

Reply:
관련 사항 적용 후 커밋 완료했습니다.

페이지 고유 id 정책에 따라 스타일링 및 레이아웃 배치가 살짝 변경되었습니다.

확인 후 승인 부탁드려요~~